pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -313,6 +304,23 @@ func (v *VPCConnectivity) getPerLayerConnectivity(layer string, src, dst Node, i
 	return result
 }
 
+// load balancer abstraction:
+// currently, AllowedConnsCombined contains the private IPs of the load balancer.
+// the abstraction creates new AllowedConnsCombined,
+// it replaces the private IPs in the with the load balancer itself
+// for each load balancer, it keeps the abstractionInfo, to be used later
+// see details at nodeSetConnectivityAbstraction()
+func (v *VPCConnectivity) abstractLoadBalancers(loadBalancers []LoadBalancer, lbAbstraction bool) {
+	if lbAbstraction {
+		abstractionInfo := newNodeSetAbstraction(v.AllowedConnsCombined)
+		for _, lb := range loadBalancers {
+			ai := abstractionInfo.abstractNodeSet(lb)
+			lb.SetAbstractionInfo(ai)
```

Comment:
can't `abstractNodeSet()` assign the abstraction info, instead of adding this call here?

Reply:
NodeSetAbstraction does not know LB